test/mock/connection.py

```
@@ -0,0 +1,47 @@
+            lm: ListenerErrorHandler, qm: QueuingErrorHandler,
+            listener: MessageQueueListener = BrokenListener(), queue: Queue = BrokenQueue(),
+    ):
+        self.lm = lm
+        self.qm = qm
+
+
+        self.listener = listener
+        self.queue = queue
+
+
+    def create(self, listener: MessageQueueListener | None = None) -> Connection:
```

Ruff (0.12.2)

Unused method argument: `listener`

(ARG002)